src/shared/components/ActionBar/ActionBar.style.ts

```diff
@@ -63,6 +63,12 @@ export const StyledButtonsContainer = styled.div`
   flex-direction: row;
   align-items: center;
   justify-content: flex-end;
+  button:hover {
+    cursor: pointer;
+  }
+  button:nth-of-type(1) {
```

Comment:
```suggestion
  > *:not(:last-child) {
```

This is a minor change but:
- by using `> *` we make sure the margin works for whatever is here, for example if we want to make the cancel button a link, this will work, `button` will not catch that
- with `nth-of-type(1)` we will add margin only to one of the buttons, if we add more of them, we'd need to update the styles. with `:not(:last-child)` we can target all items except the last one which is what I believe we want to do

Let me know if something here doesn't make sense